lib/moon_web/hooks/globals.ex

```diff
@@ -0,0 +1,16 @@
+defmodule MoonWeb.Hooks.Globals do
```

Comment:
I think Hook should have some Action name. Like "SetGlobalAssigns" etc.

Reply:
ok, will rename it a next PR